tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -289,3 +391,74 @@ def __validate_s2_records_are_related(self):
                     num_errors += 1
 
         return num_errors
+
+    def __validate_t5_aabd_and_ssi(self):
+        print('validate t5')
```

Comment:
Do we still need this? If there are other `print`'s, lets make them logs instead.